save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationSettingsMenu.kt

```diff
@@ -67,6 +72,23 @@ external interface OrganizationSettingsMenuProps : Props {
     var onCanCreateContestsChange: (Boolean) -> Unit
 }
 
+/**
+ * Makes a call to change project status
+ *
+ * @param organizationName name of the organization whose status will be changed
+ * @param status is new status
+ * @return response
```

Comment:
Please document the return type here.

```kotlin
suspend WithRequestStatusContext.() -> Response
```

is not a response &mdash; at least, you should say this is a _lazy_ response, if I understand your intention correctly.

Reply:
Done